packages/components/.storybook/preview-head.html

```diff
@@ -112,7 +112,8 @@
   .mouseless,
   .mouseless:hover {
     cursor: none !important;
-    padding: 0.5rem; /* makes focus ring completely visible in Chromatic screenshots */
+    padding: 0.5rem;
+    /* makes focus ring completely visible in Chromatic screenshots */
```

Comment:
Why is this two-lined now?

Reply:
My "Format on Save" seemed to have changed it. I will change it back since the one line passes our format verification anyways.